src/main/java/com/ericsson/ei/controller/SubscriptionControllerImpl.java

```diff
@@ -165,7 +165,7 @@
 
     @Override
     @CrossOrigin
-    @ApiOperation(value = "Removes the subscriptions from the database")
+    @ApiOperation(value = "Removes subscription(s) from the database")
```

Comment:
Maybe you can remove the "from the database" part since it's not present in the other endpoint descriptions.

Reply:
Sure, I'll remove it.